lib/qb_integration/order.rb

```diff
@@ -42,10 +52,91 @@ def cancel
       [200, text]
     end
 
+    def as_flowlink_hash(qbo_order)
+      {
+        id: qbo_order.id,
+        name: qbo_order.doc_number,
+        number: qbo_order.doc_number,
+        created_at: qbo_order.txn_date,
+        line_items: format_line_items(qbo_order.line_items),
+        currency: qbo_order.currency_ref.value,
+        customer: customer_email(qbo_order.customer_ref.value),
```

Comment:
if we added qbo order as instance var, we would be able to avoid passing this arguments around in this class, it is a bit weird to have a instance method that doesnt depend on the state of the object... but I guess qbo order could be a var in the class... or we should move it to another class... wdyt ?

Reply:
Hmm, I haven't thought about it that way before; that instance methods should depend on an instance var or state. I feel like qbo_order shouldn't be a var in this class because the main responsilibites of this class involves the payload order and passing that on to a service. It might be doing too much if it also saves the qbo version of the order